myuw/templates/base.html

```diff
@@ -550,6 +590,38 @@ <h4>What's New</h4>
             </div>
         </div>
         <!-- end modal code -->
+                
+        {% block tour-page-modal %} 
+        <!-- MyUW Tour modal -->
+        <div id="tour-page-modal"></div>
+        <div class="myuw-tour-modal-desktop myuw-modal modal fade" id="tour_modal" tabindex="-1" role="dialog" aria-labelledby="tour_modal_label" aria-describedby="tour_modal_description" aria-hidden="true" aria-modal="true">
+            <div class="modal-dialog message-dialog" role="document" style="">
+
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button id="tour_modal_close" type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                        <h3 class="modal-title" id="tour_modal_label">Welcome! Here's MyUW at a glance</h3>
+                    </div>
+                    <div class="modal-body">
+			<div style="margin-top:16px;">
+			    <picture> 
+				<source media="(min-width: 992px)" srcset="{% static '/images/myuw-tour-1.0x.png' %} 1x, {% static '/images/myuw-tour-2.0x.png' %} 2x" sizes="570px" >
+				    <source srcset="{% static '/images/myuw-tour-mobile-1.0x.png' %} 1x, {% static '/images/myuw-tour-mobile-2.0x.png' %}  2x" >
+					<img src="{% static '/images/myuw-tour-1.0x.png' %}" alt="MyUW Home page with email link, Account page, UW resources page callouts" longdesc="" style="display:block; margin-left:auto; margin-right:auto;border:1px #eee solid;">
+			    </picture>
+			</div>
+
+                        <p style="margin-top:16px;">Watch a video tour of <a href="https://www.youtube.com/watch?v=AIXJ4UPGXE4&feature=youtu.be" target="_blank" title="MyUW video tour for instructors" data-linklabel="MyUW video tour for Instructors">MyUW for Instructors</a>, <a href="" target="_blank" title="MyUW video tour for staff">for staff</a>, or <a href="https://www.youtube.com/watch?v=K7GoUc32TMs&t=5s&list=PL-hNmjMg7KSHFdXj6yXDjZtCpjkkKBLUZ&index=1" target="_blank" title="MyUW video tour for students" data-linklabel="MyUW video tour for students">for students</a>. <br><a href="https://itconnect.uw.edu/learn/tools/myuw-help-center/" target="_blank" title="MyUW Help center in IT Connect"  data-linklabel="MyUW Help center in IT Connect">Visit the MyUW help guide for more information</a>.
```

Comment:
## Link Text for help guide line

`<a href="https://itconnect.uw.edu/learn/tools/myuw-help-center/" target="_blank" title="MyUW Help center in IT Connect"  data-linklabel="MyUW Help center in IT Connect">Visit the MyUW help guide for more information</a>.`

Might suggest changing this so the link text is smaller, to make it a little easier to read. Something like:
"Visit the **MyUW help guide** for more information."

## data-linklabels

If you can, see if there are ways to shorten some of these. They don't have much space in the quick links area. "MyUW Help center in IT Connect" can probably shorten to "MyUW Help Center". I'm not sure if there's a good way to shorten the others.

Reply:
Shortened to 'MyUW Help Center'
To show more than 'MyUW video tour for instr...' the data-linklabel' is changed from 'MyUW video tour for Instructors' to 'MyUW video for Instructors'